model_drafts/tiles/edge.js

```diff
@@ -0,0 +1,29 @@
+var Sequelize = require('sequelize');
+
+function Edge(db){
+  var Edge = db.define('Edge', {
+    type: Sequelize.STRING,
+    reference: Sequelize.UUID
+  });
+  return Edge;
+}
+
+function Pusher(db){
+  var Pusher = db.define('Pusher', {
+    active: Sequelize.ARRAY(Sequelize.INTEGER) //registers on which this pusher is active. 1,3,5 or 2,4
+  });
```

Comment:
I'm not sure what the '1,3,5 or 2,4' is referring to. I think there is only one kind of pusher and pushers are mutually exclusive and can only be active on one side of the wall at a time. Will the integer be a reference to the wall it's on?